app/shared/components/Wallet/Status/Actions/Table/Row.js

```diff
@@ -0,0 +1,191 @@
+// @flow
+import React, { Component } from 'react';
+import { translate } from 'react-i18next';
+import { Icon, Table } from 'semantic-ui-react';
+import TimeAgo from 'react-timeago';
+
+import DangerLink from '../../../../Global/Modal/DangerLink';
+
+class WalletStatusActionsTableRow extends Component<Props> {
+  actionDescriptionFromAct(act) {
```

Comment:
This needs to be converted into a `type <=> component` mapping of some sort, and each different type of action should have it's own component. This huge switch statement within here is not sustainable as we grow and new operation are added.

Example:

```
{
  transfer: 'WalletStatusActionTableRowTransfer',
  delegatebw: 'WalletStatusActionTableRowDelegateBW',
  ...
}
```

The mapping can then be used to dynamically load the appropriate component and render it. 

If the action type of the action doesn't match something within the mapping, it should load a generic component that displays the raw unformatted data somehow.